src/depends/libDatabase/LevelDB.cpp

```diff
@@ -88,7 +88,7 @@ LevelDB::LevelDB(const std::string & dbName, const std::string& subdirectory, bo
     // that might not be the current directory (case when 'diagnostic' is true).
     // Its default value is false, and the non-diagnostic path is preserved
     // from the original code.
-    string db_path = diagnostic ? (m_subdirectory + PERSISTENCE_PATH) : ("./" + PERSISTENCE_PATH + (m_subdirectory.empty() ? "" : "/" + m_subdirectory));
+    string db_path = diagnostic ? (m_subdirectory + "/persistence") : (STORAGE_PATH + "/persistence" + (m_subdirectory.empty() ? "" : "/" + m_subdirectory));
```

Comment:
Would it make more sense to add `const std::string PERSISTENCE_PATH = "/persistence";` into this file so that way any edits to the path at which the DB is located can be easily updated?

Reply:
yeah. will add it.